contracts/seed/SeedFactory.sol

```diff
@@ -30,21 +30,18 @@ contract SeedFactory is CloneFactory, Ownable {
 
     event SeedCreated(address indexed newSeed, address indexed beneficiary);
 
-    /**
-     * @dev               Constructor to set the master copy address.
-     * @param _masterCopy The address of the new Seed basis.
-     */
-    constructor (Seed _masterCopy) public {
-        masterCopy  = _masterCopy;
+    modifier isInitialized {
+        require(masterCopy != Seed(0), "SeedFactory: mastercopy cannot be zero address");
+        _;
     }
 
     /**
      * @dev               Set Seed contract which works as a base for clones.
      * @param _masterCopy The address of the new Seed basis.
      */
-    function changeMasterCopy(Seed _masterCopy) public onlyOwner {
+    function setMasterCopy(Seed _masterCopy) public onlyOwner {
         require(_masterCopy != Seed(0), "SeedFactory: mastercopy cannot be zero address");
```

Comment:
1) Have to write this, because if I use argument in modifier.
"Please go to Point Number 2"

Reply:
Oh I see that it is checking `_mastercopy`, not `mastercopy`.  So maybe the message can be  "SeedFactory: _mastercopy cannot be zero address"